src/test/java/org/dita/dost/util/FilterUtilsTest.java

```diff
@@ -262,6 +262,41 @@ public void testExtCheckExclude() {
         assertFalse(f.extCheckExclude(new QName[] {PLATFORM, OS}, Arrays.asList( "unix", "amiga", "windows")));
     }
     
+    @Test
+    public void testNoInfoMessageFilter() {
+        //Do not issue information messages that certain values
```

Comment:
Could you refactor this test to something like to remove duplication and also make it easier to turn this into a parameterized test when we move to JUnit 5

```java
for (FilterKey filterKey: new FilterKey[]{ new FilterKey(OS, null), FilterUtils.DEFAULT }) {
    final Map<FilterKey, Action> fm = new HashMap<>();
    fm.put(filter, Action.EXCLUDE);
    fm.put(new FilterKey(OS, "amiga"), Action.INCLUDE);
    FilterUtils f = new FilterUtils(false, fm, null, null);
    StringBuilder infoMsg = new StringBuilder();
    f.setLogger(new TestUtils.TestLogger() {
        @Override
        public void info(String msg) {
           infoMsg.append(msg);
       }
    });

    assertTrue(f.extCheckExclude(new QName[] {OS}, Arrays.asList("osx")));
    assertEquals("Should not output info message " + infoMsg, "", infoMsg.toString());
}
```